### DISCUSSION TITLE
Subscription not working

### DISCUSSION THREAD

**Discussion by Tejesh-Duvvuru** [hasura/graphql-engine#9450] [Question]
I am trying Subscription in Hasura cloud console but it not getting data but not also throwing any error just loader icon showing. Do we need configure WebSocket or else no need for work Subscriptions

![image](https://user-images.githubusercontent.com/101654993/220335585-4ef67d75-72e5-4e02-bc96-4481de06d8ae.png)

**Comment by meetzaveri** [comment]
Hey @Tejesh-Duvvuru 

Can you check network tab -> WS -> check request with `wss://<your_url>.hasura.app/v1/graphql` -> messages ? Basically to check what error they are receiving once it has acknowledgement done. 

This will help us to troubleshoot issue further.
